src/databaseService/index.js

```diff
@@ -91,7 +91,9 @@ export const addDiagnose = async (imageReferences, text) => {
       user,
       text,
       imageReferences,
-      amountOfAnswers: 0
+      amountOfAnswers: 0,
+      removedAt: 0,
+      isSolved: false
```

Comment:
This probably would need another script to set `isSolved` in existing requests.

@nahudalla

Reply:
Yes probably in a future it will be needed if we want to query for that value. But it is not needed for this feature.